ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml

```diff
@@ -55,21 +59,42 @@ tests:
               {"step": "rosa-cluster-wait*", "failure_type": "all", "classification": "Infrastructure Provisioning - Rosa Cluster Operators", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
               {"step": "rosa-sts-account-roles-create", "failure_type": "all", "classification": "Account Roles Creation - AWS", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
               {"step": "rosa-sts-account-roles-delete", "failure_type": "all", "classification": "Account Roles Deletion - AWS", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["!default"]},
-              {"step": "rosa-conf-idp-htpasswd", "failure_type": "all", "classification": "Admin Access - Cluster", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]}
+              {"step": "rosa-conf-idp-htpasswd", "failure_type": "all", "classification": "Admin Access - Cluster", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "servicemesh-smcp-install", "failure_type": "all", "classification": "Infrastructure Provisioning - SMCP Install", "jira_project": "OSSM", "group": {"name": "lp-tests", "priority": 2}, "jira_assignee": "rcernich@redhat.com", "jira_priority": "critical"},
```

Comment:
You can also replace `"jira_assignee": "rcernich@redhat.com"` with `"jira_assignee": "!default"`. You can also replace ` "jira_priority": "critical"` with ` "jira_priority": "!default"`.

This is mostly so that you don't have to change it in multiple lines later if something changes. You would only need to change the variables that define the defaults.